app/models/forms/bulk_upload_lettings/prepare_your_file.rb

```diff
@@ -29,6 +29,10 @@ def template_path
         "/files/bulk-upload-lettings-template-v1.xlsx"
       end
 
+      def specification_path
+        "/files/Lettings 2022-23 Bulk Upload Specification.csv"
```

Comment:
I recommended replacing the spaces for underscores or hyphens

Reply:
Ah good point, forgot to put this in, cheers